springboard_iogt_ug/views.py

```diff
@@ -0,0 +1,30 @@
+from pyramid.view import view_config
+from pyramid.httpexceptions import HTTPFound, HTTPNotFound
+
+from springboard.utils import ga_context
+from springboard_iogt.views import IoGTViews, PERSONA_COOKIE_NAME, ONE_YEAR
+from springboard_iogt.utils import (
+    get_redirect_url)
+
+PERSONAE = {'YOUTH_BOY', 'YOUTH_GIRL', 'FATHER', 'MOTHER'}
+
+
+class UGIogtViews(IoGTViews):
+    @ga_context(lambda context: {'dt': 'Selected Persona', })
+    @view_config(route_name='select_persona')
+    def select_persona(self):
```

Comment:
Yeah, you can, with a little bit more effort. The issue is often that a class method's first argument is `self` so for non-class-method decorators, you can't just use them as is. If you're writing your own, you can easily cater for that.